spec/fabricators/bgeigie_import_fabricator.rb

```diff
@@ -1,4 +1,4 @@
 Fabricator(:bgeigie_import) do
-  source File.new(Rails.root.join("spec/fixtures/bgeigie#{Fabricate.sequence}.log"))
+  source { sequence(:source) { |n| Rails.root.join("spec/fixtures/bgeigie#{n % 6}.log").open(File::RDONLY) } }
```

Comment:
Gotcha. I tend to see a fair bit of `File.new(pathname_object)` but seems like either is basically the same.

As an aside I thought maybe `File.new` doesn't open file handlers, but looks like it does. TIL